go/cmd/vtbench/vtbench.go

```diff
@@ -29,7 +29,9 @@ import (
 	"vitess.io/vitess/go/vt/logutil"
 	"vitess.io/vitess/go/vtbench"
 
+	// Imports and register the gRPC vtgateconn client
```

Comment:
Should be `Import and register...`
Can you fix below also?

Reply:
updated.